spec/rubocop/cop/style/struct_inheritance_spec.rb

```diff
@@ -46,4 +46,44 @@ def age
       end
     RUBY
   end
+
+  it 'autocorrects simple inline class with no block' do
+    new_source = autocorrect_source(<<~RUBY)
+      class Person < Struct.new(:first_name, :last_name)
+      end
+    RUBY
+
+    expect(new_source).to eq(<<~RUBY)
+      Person = Struct.new(:first_name, :last_name)
+    RUBY
+  end
+
+  it 'autocorrects a class with an empty block' do
+    new_source = autocorrect_source(<<~RUBY)
+      class Person < Struct.new(:first_name, :last_name) do end
+      end
+    RUBY
+
+    expect(new_source).to eq(<<~RUBY)
+      Person = Struct.new(:first_name, :last_name) do end
+    RUBY
+  end
+
+  it 'autocorrects a class with a block' do
```

Comment:
Hmmm.  Looking at the docs for `Struct.new` it's not clear to me what the first (empty) block is doing.  It looks like that code shouldn't even parse but it definitely does.  I think I'm missing something because I thought the only block `Struct.new` took *was* the class body.

Is the block of the class body actually part of the `Class` node and this empty block a parameter to `Struct.new`?